doc/python/imshow.md

```diff
@@ -399,6 +399,95 @@ for compression_level in range(0, 9):
 fig.show()
 ```
 
+### Exploring 3-D images, timeseries and sequences of images with `facet_col`
+
+*Introduced in plotly 4.14*
+
+For three-dimensional image datasets, obtained for example by MRI or CT in medical imaging, one can explore the dataset by representing its different planes as facets. The `facet_col` argument specifies along which axis the image is sliced through to make the facets. With `facet_col_wrap`, one can set the maximum number of columns. For image datasets passed as xarrays, it is also possible to specify the axis by its name (label), thus passing a string to `facet_col`.
+
+It is recommended to use `binary_string=True` for facetted plots of images in order to keep a small figure size and a short rendering time.
+
+See the [tutorial on facet plots](/python/facet-plots/) for more information on creating and styling facet plots.
+
+```python
+import plotly.express as px
+from skimage import io
+from skimage.data import image_fetcher
+path = image_fetcher.fetch('data/cells.tif')
+data = io.imread(path)
+img = data[20:45:2]
+fig = px.imshow(img, facet_col=0, binary_string=True, facet_col_wrap=5)
+fig.show()
+```
+
+Facets can also be used to represent several images of equal shape, like in the example below where different values of the blurring parameter of a Gaussian filter are compared.
```

Comment:
I added this example @nicolaskruchten to show that sequences of images can also be represented with facets. Should we accept lists and call `np.array` on lists or tuples at the beginning of `px.imshow`? I have mixed feelings about this, because not any list will work (in particular arrays of different shapes) so if there is an error to be thrown I'd rather have it thrown in the user's code than by `imshow`.

Reply:
let's let the user deal with it.

Reply:
what do you think about how facet titles are set in this example? Should we introduce a parameter for facet titles in imshow?

Reply:
`labels=dict(facet_col="...")` is the general PX pattern here, yes, if you pass in an array to `facet_col=array`, and it shows up as `facet_col=<value>`. I would like to keep the idea of "if you see `x=` you can relabel it with `labels=dict(x="new")` which isn't the case with the default `"slice"` right now.